pages/inspector/connect-inspector-to-gtm.mdx

```diff
@@ -1,14 +1,74 @@
-# Send data from Google Tag Manager to Avo Inspector
+import { Callout } from 'nextra/components';
 
-## Use Avo Inspector as a GTM tag
+# Google Tag Manager Source (beta)
 
-You can use a server side GTM to analyze your tracking with Avo Inspector.
+_This document describes how to send data from Google Tag Manager to Avo Inspector._
 
-We provide a tag template to import, which you can use to create a tag and connect it to your GTM clients.
+<Callout type="info">
+The Inspector Google Tag Manager integrations are currently in beta so please reach out to us at support@avo.app or in the chat bubble in the bottom right corner if you have any feedback.
+</Callout>
 
-> Avo Inspector operates on the environment level, please keep it in mind and create separate tags for development, staging and production data.
+## Destination Info
 
-## Guide to connect GTM and Avo Inspector
+* Supports both client side and server side Google Tag Manager  
+* Inspector events can be verified using the browser developer console
+* Observability can be added to all data layer events using client side GTM in under 90 seconds
+* Observability can be added to all Google Analytics events using our server side GTM installation option
+* Events detected by Inspector can be added to the Avo tracking plan within the Inspector UI, allowing you to fill a documentation gap on legacy events in seconds
+
+
+Google Tag Manager (GTM) is a tag management system that allows you to quickly update tags and code snippets on your website, web apps, and apps. Once you deploy the Tag Manager snippet, you can configure (trigger) tags using a web-based user interface without having to alter and deploy additional code. It is common for early stage companies to use Google Tag Manager to take care of dispatching their core analytics events to reduce developer dependencies whenever new events are needed, and as the analytics maturity grow and more accuracy is needed, developers will deploy in-code data layer events, and then GTM tag owners will trigger the analytics service events off the data layer events accordingly. [Learn more about that here](https://www.loom.com/share/22c9e0f1175141efa7a0bdbf51201096). 
+
+**_Reminders:_** 
```

Comment:
@thoragudf can we make in one of those colour highlighted boxes like we have done elsewhere in this doc, or you think that's a worst experience?